kork/kork-github/kork-github.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: "java-library"
+apply from: "$rootDir/../kotlin.gradle"
+apply from: "$rootDir/gradle/detekt.gradle"
+apply from: "$rootDir/gradle/lombok.gradle"
+
+dependencies {
+  api(platform(project(":spinnaker-dependencies")))
+
+  // hub4j/github-api for GitHub interactions
+  api "org.kohsuke:github-api:1.321"
+
+  // JWT for GitHub App authentication
+  implementation "io.jsonwebtoken:jjwt-api:0.11.5"
+  implementation "io.jsonwebtoken:jjwt-impl:0.11.5"
+  implementation "io.jsonwebtoken:jjwt-jackson:0.11.5"
+
+  // Bouncy Castle for PEM parsing
+  implementation "org.bouncycastle:bcpkix-jdk15on:1.70"
```

Comment:
NIT: https://github.com/spinnaker/spinnaker/blob/main/kork/spinnaker-dependencies/spinnaker-dependencies.gradle#L12 version is defined here, shouldn't specify 1.70 - it SHOULD replace via the platform platform definition up above.  Still better to NOT explicitly set it IMHO...